ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -187,6 +187,18 @@ tests:
     - ref: ovn-sdn-migration-rollback
     - ref: openshift-e2e-test
     workflow: openshift-e2e-aws
+- as: e2e-hypershift
+  cluster_claim:
+    architecture: amd64
+    as: this-is-the-name-of-the-cluster-claim-release-which-defaults-to-latest-i-dont-want-that
```

Comment:
:thinking:

Reply:
It literally needs to have any value that is not `latest`, since we can't add comments I thought I'd use the value to document its purpose :upside_down_face: Open to change it, though.